_episodes/20-parallel-raster-computations.md

```diff
@@ -0,0 +1,347 @@
+---
+title: "Parallel raster computations using Dask"
+teaching: 40
+exercises: 20
+questions:
+- "How can I parallelize computations on rasters with Dask?"
+- "How can I determine if parallelization improves calculation speed?"
+- "How can I control how to parallelize calculations?"
```

Comment:
It is not clear to me what is meant by the last question. Can we rephrase it to something like: `What are the good practices in applying a parallelization approach?`

Reply:
Good point, but maybe that's too general? How about:

"What are good practices in applying parallelization to my raster calculations?"